### ISSUE TITLE
Error 'Invalid configuration: healthChecks.ps1' in 4.2.0 (6637675699)

### ISSUE THREAD

**Issue by ChrisNe11** [ironmansoftware/powershell-universal#2798]
### Version

Other Please Specify

### Severity

Low

### Steps to Reproduce

We have been having issues with running Apps that depend on the $Job = Invoke-PSUScript since upgrading to 4.1.7. We have not managed to get this going despite trialling different ways as discussed in the forum etc.. After seeing that 4.2.0 contains lots of fixes for the Invoke-PSUScript issue we upgraded. Since then Apps get stuck in 'Starting' and we see "Error 'Invalid configuration: healthChecks.ps1'" under Info bar. This might be a new bug introduced with this nightly version?

### Expected behavior

```console
App start up automatically as configured
```

### Actual behavior

```console
Apps get stuck at Starting. "Error 'Invalid configuration: healthChecks.ps1'"
```

### Additional Environment data

4.2.0 (6637675699)

### Visuals

<img width="498" alt="PSU - Error HealthCheck" src="https://github.com/ironmansoftware/issues/assets/148976649/e2902753-f016-4e90-a963-b55b364104ac">

**Comment by adamdriscoll** [comment]
@ChrisNe11 - Invoke-PSUScript has been fixed in 4.1.8 nightly. I can take a look at this issue in v4.2.

**Comment by ChrisNe11** [comment]
@adamdriscoll - Thx a lot. We will hang on for a little bit longer before downgrading to 4.1.8 in case this one not a big issue. Forgot to mention that the Environment is PS 7.3.6 with a persistent SQL DB.
